README.md

```diff
@@ -1,3 +1,3 @@
-This is a minimal example of a book based on R Markdown and **bookdown** (https://github.com/rstudio/bookdown). Please see the page "Get Started" at https://bookdown.org/ for how to compile this example.
+This is a minimal example of a book based on R Markdown and **bookdown** (https://github.com/rstudio/bookdown). Please see the page "[Get Started](https://bookdown.org/yihui/bookdown/get-started.html)" at https://bookdown.org/yihui/bookdown/ for how to compile this example.  Details on GitHub configuration can be found on the "[GitHub](https://bookdown.org/yihui/bookdown/github.html)" page.
```

Comment:
Why is it necessary to mention the Github page here? Personally I don't quite recommend publishing via Github pages.

Reply:
This relates to the code in https://github.com/rstudio/bookdown-demo/blob/master/_deploy.sh -- I was trying to remember how to configure this in a new repository, so came to this repo to remind myself.

Reply:
That is a too specific technical detail for this brief README. I think you can bookmark the page in your browser if it is important. If we have to provide a link somewhere, we can add it in `_deploy.sh` in a comment.